### DISCUSSION TITLE
Smart connections font size and colour

### DISCUSSION THREAD

**Discussion by FamedBear16** [brianpetro/obsidian-smart-connections#375] [Ideas]
Hi,

Is there any way to change the smart connection size and colour in the connection panel?

I cannot see very well with my glasses and to read small red text on black is very difficult. 

It would be great if there is a way to customise it in the interface or a quick solution for the moment.

Thanks

**Comment by jowitte** [comment]
Adding up on this: Actually the chat windows is unreadable in my theme, see screenshot. I also would like to be able to change the font colour for the Smart Connections Files list...

<img width="439" alt="Xnip 2024-03-11T074405" src="https://github.com/brianpetro/obsidian-smart-connections/assets/8180200/f8291e3e-2653-4cbc-b861-a1a47acc8b69">

**Reply by FamedBear16** [reply]
@jowitte For the smart list I use this snippet. The default colour and font were too small for my eyesight.
.sc-list .search-result small {
  color: yellowgreen;
  font-size: 1rem;
  /* color: var(--h5-color); */
  /* font-size: 0.8rem; */
  font-weight: 500;
}

**Reply by jowitte** [reply]
@FamedBear16 Thanks! Got it. I added some customisation to the chat as well:

```
.sc-message.user {
   background-color: var(--color-yellow);
   color: var(--color-black);
   font-size: 1 rem;
   font-weight: 300;
}

.sc-message.assistant {
   background-color: var(--color-yellow);
   color: var(--color-black);
   font-size: 1 rem;
   font-weight: 300;
}
```

**Reply by brianpetro** [reply]
@jowitte if you update to the latest version, the chat styles have been improved as seen here https://github.com/brianpetro/obsidian-smart-connections/pull/491#issuecomment-1971702971

🌴

**Reply by jowitte** [reply]
@brianpetro Thanks. I'm using Atom and the font colour seems to make issues:

**Obsidian Atom**
<img width="425" alt="Xnip 2024-03-16T160617" src="https://github.com/brianpetro/obsidian-smart-connections/assets/8180200/e00d2c0d-cd71-4990-943a-b1552f43a402">
<img width="422" alt="Xnip 2024-03-16T160603" src="https://github.com/brianpetro/obsidian-smart-connections/assets/8180200/15226b4f-ac6e-4511-8c4f-338919f4497f">

**Reply by brianpetro** [reply]
@jowitte try updating to the latest version. There were some recent style updates thanks to @sofvanh https://github.com/brianpetro/obsidian-smart-connections/pull/491#issuecomment-1971702971

**Reply by jowitte** [reply]
The screenshots have been made after the latest update.

**Reply by brianpetro** [reply]
@jowitte, thanks for bringing this to my attention. It should be fixed in `v2.0.136` 🌴

![Screenshot 2024-03-16 at 11 36 58 AM](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/36895559-7bbc-4b70-b129-ea4bf6422fdd)
![Screenshot 2024-03-16 at 11 38 07 AM](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/b05cd6b4-bed6-4698-868a-d24ffd7d6877)

**Reply by sofvanh** [reply]
(AFAIK my style updates haven't been merged yet)

**Reply by brianpetro** [reply]
@sofvanh while I haven't officially merged the pull request yet, I did pull out most of the suggestions and put them in the stylesheet in the latest version.

**Reply by brianpetro** [reply]
@sofvanh I just realized your #491 PR was for the v1 stylesheet (realized after merging)

The V2 stylesheet is very similar and is located here https://github.com/brianpetro/obsidian-smart-connections/blob/main/src/styles.css
